README.md

```diff
@@ -18,26 +16,52 @@ If you want to test the apps interaction with third party services (e.g. GitHub
 and GOV.UK Notify) you'll need to set up some secrets in environment variables.
 The easiest way to do this in development is using a `.env` file (see [.env.example](.env.example)):
 
-```
+```shell
 cp .env.example .env
 $EDITOR .env
 ```
 
 Run the tests with:
 
-```
+```shell
 bundle exec rails test
 ```
 
 To sign in as a development user, visit <http://localhost:3000/dev-login> (to try different email addresses, you can provide a `email` parameter). If you want to test with real Google SSO, you can [create an application in the Google Cloud Console](https://console.developers.google.com/apis/credentials).
 
-Deploying to PaaS
------------------
+## Building the Docker image
+
+In order to move this app away from GOV.UK PaaS, we have containerised the 
+application in order to be able to run locally and also deploy to AWS
+App Runner. The app can be accessed on port 3000.
+
+To build and run the Docker image:
+
+```shell
+docker build -t request-an-aws-account .
+docker run -p 3000:3000 -t request-an-aws-account
+```
+
+## Deploying through GitHub Actions
+
+We use a GitHub Action to automate the building and deploying of our Docker image to GHCR. This workflow runs every time a PR is raised to the branch called 'main'. It uses the `docker/login-action`, `docker/metadata-action`, and `docker/build-push-action`.
+
+You can view the workflow in action [here](https://github.com/alphagov/re-request-an-aws-account/actions/runs/7816113717).
```

Comment:
Link to the YAML file, rather than a run. Also, it's good practice to avoid linking just the word "here"; our eyes are drawn to the link, so it is better to make it a meaningful word or phrase: for example `You can [view the workflow at publish-docker-image.yaml](...)`